plugins/updater/Cargo.toml

```diff
@@ -32,6 +32,10 @@ tar = "0.4"
 [target."cfg(target_os = \"windows\")".dependencies]
 zip = { version = "0.6", default-features = false }
 
+[target."cfg(target_os = \"macos\")".dependencies]
+flate2 = "1.0.27"
```

Comment:
the `zip` crate has `flate2` feature, could we use that instead of adding a new dependencies entry?